Sources/MintCLI/Commands/UninstallCommand.swift

```diff
@@ -5,6 +5,7 @@ import SwiftCLI
 class UninstallCommand: MintCommand {
 
     @Param var package: String
+    @Key("-v", "--version", description: "Specify the version to uninstall") var version: String?
```

Comment:
Has this been tested? I think you'll need to pass this into the uninstall call below

Reply:
Good point, thanks for catching that.
I had added the `--version` flag to the command but forgot to actually pass it down to the `Mint.uninstall` call. I’ll update the PR so the version is properly forwarded and the uninstall behaves as expected.